<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalPage</class>
 <widget class="QWidget" name="JournalPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>748</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>839</width>
        <height>728</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>28</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Food Journal</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MealLog" name="meal1" native="true"/>
       </item>
       <item>
        <widget class="MealLog" name="meal2" native="true"/>
       </item>
       <item>
        <widget class="MealLog" name="meal3" native="true"/>
       </item>
       <item>
        <widget class="MealLog" name="meal4" native="true"/>
       </item>
       <item>
        <widget class="MealLog" name="meal5" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MealLog</class>
   <extends>QWidget</extends>
   <header>items/MealLog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
